lib/enumerations/value.rb

```diff
@@ -16,6 +16,10 @@ def to_sym
       symbol
     end
 
+    def quoted_id
```

Comment:
Dont think this quoted id works as i initally thought. Remove this from this PR, and please add it to a separate issue like https://github.com/infinum/enumerations/issues/20